src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -125,8 +221,19 @@ const PastAnalysisTable = () => {
       row: { isExpanded, cells },
       ...props
     } = tableRow;
+
     const isAnalyzing: boolean = cells[4] && cells[4].title; // 4 is the last index in row
-    const backgroundStyle = { 'backgroundColor': `${isAnalyzing ? '#F9E0A2' : '#FFFFFF'}` };
+    
+    // Style the current row
+    let backgroundStyle = {};
```

Comment:
What is the use of `backgroundStyle` here, if nothing is using it after assigning its appropriate value?